.circleci/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:16.04
+MAINTAINER Gruntwork <info@gruntwork.io>
+
+# Install Bats
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+    apt-get install -y software-properties-common && \
+    add-apt-repository ppa:duggan/bats && \
+    DEBIAN_FRONTEND=noninteractive apt-get update && \
+    apt-get install -y bats
```

Comment:
Very nice! How did you like using bats?

Reply:
I've used it a couple times before. For the most part, it's decent. It's easy to learn and works more or less as you'd expect. The only downside is when debugging. In part, that's just bash itself, with it's tons of gotchas, but bats adds some of its own magic that gets in the way.